app/src/main/java/org/mozilla/fenix/components/Components.kt

```diff
@@ -207,7 +208,10 @@ class Components(private val context: Context) {
                     core.pocketStoriesService,
                     context.pocketStoriesSelectedCategoriesDataStore,
                 ),
-                MessagingMiddleware(messagingStorage = analytics.messagingStorage),
+                MessagingMiddleware(
```

Comment:
Unless I've misunderstood, are we also going to require:
`MessagingMiddleware(
                    surface = MessageSurfaceId.NOTIFICATION,
                    messagingStorage = analytics.messagingStorage
                ),`
?